### ISSUE TITLE
Arm Ethos-U85: Add support for BatchNorm2d not immediately following a convolution

### ISSUE THREAD

**Issue by itaiberman** [pytorch/executorch#17241]
### 🚀 The feature, motivation and pitch

I'm trying to export a model that has a BatchNorm2d layer without a Conv layer before that and its not supported. This is the model's [repository](https://github.com/Talented-Q/POSTER_V2/tree/main).
Supporting this should be simple given that BatchNorm2d can just be broken down to mul and add
The reason I get in the log is: "BatchNorm2D with track_running_stats==True not immediately following a convolution is not supported for quantized TOSA backends."
If I change the layer to have `track_running_stats=False` its still not supported with reason: "Not included in BaseTOSASupportList or a registered tosa_support_check".

### Alternatives

I managed to substitute the BatchNorm2d to a module that does the exact same thing and it allows export.
```
class CustomBatchNorm2d(nn.Module):
    def __init__(self, weight, bias, running_mean, running_var):
        super(CustomBatchNorm2d, self).__init__()
        self.weight = weight.view(1, -1, 1, 1)
        self.bias = bias.view(1, -1, 1, 1)
        self.running_mean = running_mean.view(1, -1, 1, 1)
        self.var = torch.sqrt(running_var + 1e-5).view(1, -1, 1, 1)

    def forward(self, x):
        x_normalized = (x - self.running_mean) / self.var
        out = self.weight * x_normalized + self.bias

        return out
```

cc @freddan80 @per @zingo @oscarandersson8218 @digantdesai

**Comment by nil-is-all** [comment]
Hi @per @zingo, could a model that has a BatchNorm2d layer without a Conv layer be supported?

**Comment by tom-arm** [comment]
Hi @itaiberman, thank you for the feature request, we can put this into our backlog.

It's good to see there is a workaround for this - does this workaround completely unblock you?

**Comment by nil-is-all** [comment]
Hey @tom-arm, I'm part of the PyTorch Edge triage team; wanted to bring this feature request from @itaiberman to your attention. Do tag any working PRs if any for this issue. Thanks!

**Comment by tom-arm** [comment]
> Hey [@tom-arm](https://github.com/tom-arm), I'm part of the PyTorch Edge triage team; wanted to bring this feature request from [@itaiberman](https://github.com/itaiberman) to your attention. Do tag any working PRs if any for this issue. Thanks!

Ah my bad! I've modified my message to refer to the issue author

**Comment by itaiberman** [comment]
Hey @tom-arm , the workaround fixes one of the problems with the specific model and now it no longer lists it as unsupported. Created another [issue](https://github.com/pytorch/executorch/issues/16902) regarding another unsupported PReLU which i also managed a workaround.

**Comment by itaiberman** [comment]
Hi @tom-arm , I also found that Ethos-U doesn't have support for BatchNorm1d. Since the fix is similar, I don't want to open another issue.

**Comment by nil-is-all** [comment]
Hi @tom-arm, just wanted to ask if there was an update on support for BatchNorm2d. Is there any PR in progress to tag here?

**Comment by tom-arm** [comment]
Sorry for the delayed response here. No update yet on this issue. @itaiberman is it still the case you can work around this?

**Comment by itaiberman** [comment]
@tom-arm yes, the workaround I used in my original message works

**Comment by nil-is-all** [comment]
Hi @tom-arm, any update you could provide on BatchNorm2D support?

**Comment by tom-arm** [comment]
Hi @nil-is-all, apologies for the late reply. A solution is being worked on, I will reply to this issue when I have a PR ready

**Comment by tom-arm** [comment]
With https://github.com/pytorch/executorch/pull/21710 being merged, this issue should be resolved now

**Comment by nil-is-all** [comment]
> With [#21710](https://github.com/pytorch/executorch/pull/21710) being merged, this issue should be resolved now

Thanks @tom-arm. @itaiberman closing this issue as resolved, feel free to re-open if it persists.
